mne/cov.py

```diff
@@ -386,66 +389,90 @@ def compute_raw_covariance(raw, tmin=None, tmax=None, tstep=0.2,
     picks : array-like of int | None (default None)
         Indices of channels to include (if None, all channels
         except bad channels are used).
+    keep_sample_mean : bool (default False)
```

Comment:
From the 2.7.3 manual:
```
If the recommended keepsamplemean option is specified in the covariance
matrix definition file, the baseline correction is applied to the epochs but
the means at individual samples are not subtracted.
```
So I think the problem is that the computation was done with the recommended option `keepsamplemean` turned off. In other words, the file in our repo is wrong, and our code has had a bug (bad default behavior) for a long time. I'll change it here and change `whats_new.py`. Thanks for slogging through this with me y'all.